src/options/ConfigOptions.ts

```diff
@@ -31,7 +31,7 @@ type Options = Partial<{
   cherrypickRef: boolean;
   commitConflicts: boolean;
   commitPaths: string[];
-  copySourcePRLabels: boolean;
+  copySourcePRLabels: boolean | string[];
```

Comment:
I wasn't able to figure out a way to make this change in `cliArgs.ts`. We could also define a new option instead of allowing two types in the existing option, if that's preferred. Let me know what you think.